src/store/root.store.ts

```diff
@@ -47,14 +46,15 @@ export default class RootStore {
   newsFlashLoading: boolean = false;
   widgetBoxLoading: boolean = false;
   currentLanguageRouteString: string = '';
-  selectedLanguage: string = LANG.HE;
+  selectedLanguage: string = 'he';
```

Comment:
This change is not required. moreover - it against the purpose of `LANG` enum
keep the `LANG` enum as is